### PR TITLE
Tooltip: minor refactor

### PR DESCRIPTION
**Pull request by AlbertCarreras**
#### What changed?

Tooltip: minor refactor

### CODE REVIEW

File: packages/gestalt/src/Contents.js [hunk 1]
```diff
@@ -213,51 +213,48 @@ class Contents extends Component<Props, State> {
 
     return (
       <div
-        className={styles.container}
```
Comment (AlbertCarreras) [inline packages/gestalt/src/Contents.js:216]: Merge this and the following div into 1

File: docs/pages/scrollboundarycontainer.js [hunk 2]
```diff
@@ -121,48 +121,49 @@ function ScrollBoundaryContainerExample() {
               idealDirection="right"
               text="See stats about different types of content created by you and/or others on Pinterest. Filter to get more details on your organic (not an ad) and paid (promoted as an ad) content."
             >
-              <Icon
```
Comment (AlbertCarreras) [inline docs/pages/scrollboundarycontainer.js:124]: Example better a11y

File: docs/pages/scrollboundarycontainer.js [hunk 3]
```diff
@@ -121,48 +121,49 @@ function ScrollBoundaryContainerExample() {
...
-                checked={content === "paid"}
-                id="paid"
-                label="Paid and earned"
-                name="content"
-                onChange={() => setContent( "paid" )}
-                value="paid"
-              />
-            </Box>
-          </Box>
+          <Flex direction="column">
```
Comment (dangerismycat) [inline docs/pages/scrollboundarycontainer.js:133]: looks like the left side is getting cut off a bit

Screen Shot 2022-05-09 at 3 38 10 PM

File: docs/pages/scrollboundarycontainer.js [hunk 4]
```diff
@@ -121,48 +121,49 @@ function ScrollBoundaryContainerExample() {
...
+            <Fieldset legend="Select content type" legendDisplay="hidden">
+              <Box paddingY={1}>
```
Comment (dangerismycat) [inline docs/pages/scrollboundarycontainer.js:135]: should this use Flex and `gap`, or does that not work well with Fieldset?

**Review by dangerismycat — APPROVED** [review]

File: docs/pages/scrollboundarycontainer.js (hunk 3 above)
Comment (AlbertCarreras) [reply docs/pages/scrollboundarycontainer.js:133]: Cant repro this

File: docs/pages/scrollboundarycontainer.js (hunk 4 above)
Comment (AlbertCarreras) [reply docs/pages/scrollboundarycontainer.js]: DONE. good catch

**Pull request merged**